other/io.github.antimicrox.antimicrox.yml

```diff
@@ -0,0 +1,28 @@
+app-id: io.github.antimicrox.antimicrox
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: antimicroX
```

Comment:
Is it command invoking antimicrox?
If yest I think `antimicrox` would be better
https://github.com/AntiMicroX/antimicroX/tree/package_renaming

Reply:
I see now it is executable name, let it be as it is, it will be changed later